articles/role-based-access.md

```diff
@@ -105,6 +105,8 @@ GitOps is an API-only and write-only role that can be used on CI/CD pipelines.
 
 \** Applies only to [Fleet REST API](https://fleetdm.com/docs/using-fleet/rest-api)
 
+\*** GitOps users do not have access to modify `gitops` settings.
```

Comment:
Hey @rachaelshaw as someone looking at this for the first time (new user), I'm not sure I understand what `gitops` settings means. It's about editing GitOps mode right? Maybe we add a new "Edit GitOps mode" row instead?

That said, I thought we weren't adding any permissions changes as part of [the user story](https://github.com/fleetdm/fleet/issues/25478). Did that change?

![Screenshot 2025-03-04 at 9 30 33 AM](https://github.com/user-attachments/assets/dc2ad7b9-c42c-4975-8519-93ee672c7f87)

Reply:
@noahtalerman ah I see what you mean, we should still allow the GitOps _role_ to modify GitOps mode org settings (if someone wanted to do that via the API for some reason), but just not allow it to be updated via YAML.  I was conflating the GitOps role with the `fleetctl gitops` command in my mind.

I'll close this.